apps/.storybook/config.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import * as storybook from '@kadira/storybook';
+import centered from '@kadira/react-storybook-decorator-centered';
+import infoAddon from '@kadira/react-storybook-addon-info';
+import addStoriesGroup from 'react-storybook-addon-add-stories-group';
+import {setExternalGlobals, setupLocale} from '../test/util/testUtils';
+import color from '../src/color';
+setExternalGlobals();
+
+function addStyleguideExamples(subcomponent) {
+  if (subcomponent && subcomponent.styleGuideExamples) {
+    subcomponent.styleGuideExamples(
+      storybook,
+      {centered}
+    );
+  }
+}
+
+function loadStories() {
+  storybook
+    .storiesOf('Colors', module)
+    .add('yay', () => (
+      <div>
+        <h2>Colors</h2>
+        <p>
+          These are the standard colors available in SCSS and javascript.
+        </p>
+        <p>
+          JavaScript usage is as follows:
+        </p>
+        <pre>
+          {`import colors from '@cdo/apps/colors';
+console.log("the hex value for purple is:", colors.purple);`}
+        </pre>
+        {Object.keys(color).map(colorKey => (
+           <div key={colorKey} style={{display: 'inline-block',
+                                       whiteSpace: 'nowrap',
+                                       width: '33%',}}>
+             <div style={{display: 'inline-block',
+                          border: '1px solid black',
+                          width: 50,
+                          height: 50,
+                          backgroundColor: color[colorKey]}}/>
+             <div style={{display: 'inline-block',
+                          position: 'relative',
+                          bottom: 12.5,
+                          paddingLeft: 10,}}>
```

Comment:
Our style preference elsewhere is to have a single `styles` object at the top of the file instead of a bunch of inlined objects